toxbat/requirements.py

```diff
@@ -122,11 +115,10 @@ def is_changed(fpath, prev_version_fpath):
     if not (fpath and os.path.isfile(fpath)):
         raise ValueError("Requirements file {0!r} doesn't exist.".format(fpath))
 
-    # Compile the list of new requirements.
-    new_requirements = parse_pip_requirements(fpath)
-
-    # Hash them.
-    new_requirements_hash = _str_to_sha1hex(str(new_requirements))
+    # Hash the requirements file.
+    with open(fpath, 'r') as req_file:
+        # Hash them.
+        new_requirements_hash = _str_to_sha1hex(req_file.read())
```

Comment:
I think it might be useful to ignore empty lines and and comments + ignore the order. 
It's a common case when pre-commit sorts the entries in the file.
I can do it in a separate PR.